_blueprints/developers_guide.adoc

```diff
@@ -3,15 +3,16 @@
 == Adding new packages
 
 ALMighty uses https://glide.sh[Glide] for package management.  To
-install a new package, you can follow these steps.  The `<URL>` should
-be replaced with the package URL pointing to Github or any other
-location.
+install a new package, you can follow these steps.  The
+`<IMPORT_PATH>` should be replaced with the actual import path of the
```

Comment:
> The link is taking me to Github.

You should see this:

![screenshot from 2016-09-01 09-21-31](https://cloud.githubusercontent.com/assets/193408/18158765/8688472c-7025-11e6-8eab-e1c593c85b89.png)

> Anyway, I have changed it to "package path" now. Thanks for the review!

Great!





> Anything under vendor is still imported without the vendor prefix, right?

Right, if a package is stored on the file system under `$GOPATH/src/github.com/almighty/almighty-core/vendor/github.com/foo/bar` you can import it as `github.com/foo/bar`.